centaur/src/main/scala/centaur/serialization/TestsReports.scala

```diff
@@ -0,0 +1,46 @@
+package centaur.serialization
+
+import java.io.{FileInputStream, FileOutputStream}
+
+import com.esotericsoftware.kryo.io.{Input, Output}
+import com.twitter.chill.ScalaKryoInstantiator
+
+import scala.collection.mutable
+
+/**
+  * This class contains information about last version in which corresponding integration test was successfully run
+  * @param testCasesRuns  key is a test name; value is a version of Cromwell, in which this test passed last time
+  */
+case class TestsReports(testCasesRuns: scala.collection.mutable.Map[String, String]) {
```

Comment:
A mutable data structure is almost always not the best choice. Please consider using `TrieMap`.